i18n/en/docusaurus-theme-classic/footer.json

```diff
@@ -9,11 +9,11 @@
   },
   "link.item.label.Discord": {
     "message": "Discord",
-    "description": "The label of footer link with label=Discord linking to https://discord.gg/K5THjAweFB"
+    "description": "The label of footer link with label=Discord linking to https://discord.gg/camino"
   },
   "link.item.label.Twitter": {
     "message": "Twitter",
-    "description": "The label of footer link with label=Twitter linking to https://twitter.com/CaminoFndtn"
+    "description": "The label of footer link with label=Twitter linking to https://twitter.com/caminonetwork"
   },
   "link.item.label.Telegram": {
     "message": "Telegram",
```

Comment:
Shouldn't this also be changed to camino_network in line 20? 
Please double check if you got every link right.